common/network-common/src/main/java/org/apache/spark/network/util/TransportFrameDecoder.java

```diff
@@ -184,8 +184,12 @@ private ByteBuf decodeNext() {
       return null;
     }
 
-    // Reset buf and size for next frame.
+    return comsumeCurrentFrameBuf();
```

Comment:
consume

Reply:
ups typo... I blame my fat fingers :) Let me fix this.